spaces/domains.json

```
@@ -381,6 +381,8 @@
   "dao.babyspacefloki.com": "babyspacefloki.eth",
   "vote.meowfinance.org": "meowfinance.eth",
   "gov.dot.finance": "dotfinance.eth",
+  "vote.nftvote.net": "nftvote.eth",
+  "vote.mgh.dao": "metagamehub.eth"
   "voting-v2.multiplier.finance": "v2-multiplierfinance.eth",
```

A comma is missing at the end of the line below your sub-domain.
```suggestion
  "vote.nftvote.net": "nftvote.eth",
  "vote.mgh.dao": "metagamehub.eth",
  "voting-v2.multiplier.finance": "v2-multiplierfinance.eth",
```